home/tests/test_insert_question_feature.py

```diff
@@ -125,26 +127,30 @@ def test_post_valid_question_with_client(self, client, valid_data):
             # User entered all the required fields and to high book page
         ])
         @pytest.mark.django_db
-        def test_post_invalid_question_with_client(self, client, invalid_data):
-            client.login(username='Lior', password='LiorLior')
+        def test_post_invalid_question_with_client(self, client, invalid_data, authenticated_user):
             client.post('/explore/new_question', data=invalid_data)
```

Comment:
Its not directly related to this PR - but it seems that assertions for the HTTP response are missing here and in other similar tests. How do you know the right error message is shown to the user?